lib/match/manual.rb

```diff
@@ -0,0 +1,100 @@
+require 'securerandom'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.